src/System.Management.Automation/engine/SessionStateContainer.cs

```diff
@@ -3689,10 +3690,16 @@ internal Collection<PSObject> NewItem(string[] paths, string name, string type,
 
             foreach (string path in paths)
             {
+                string resolvePath = null;
                 if (path == null)
                 {
                     PSTraceSource.NewArgumentNullException("paths");
                 }
+                else
+                {
+                    // To be compatible with Linux OS.
```

Comment:
Should we specified in the comment that Path.AltDirectorySeparatorChar referred to  ('\') on UNIX and a slash ('/') on Windows and Mac?